experimental/plugins_graphdriver.md

```diff
@@ -269,7 +269,7 @@ Get a list of changes between the filesystem layers specified by the `ID` and
 }
 ```
 
-Responds with a list of changes. The structure of a change is:
+Respond with a list of changes. The structure of a change is:
```

Comment:
No, I think this document is just not very consistent. Experimental docs usually don't go through a full docs-review, as they may still change